pom.xml

```diff
@@ -361,6 +340,13 @@
         <version>3.48.0</version>
         <scope>import</scope>
       </dependency>
+      <dependency>
+        <groupId>org.junit</groupId>
+        <artifactId>junit-bom</artifactId>
+        <version>${junit.version}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
```

Comment:
This effectively replaces the need for individual junit dependency blocks that have been removed.